CHANGES.md

```diff
@@ -247,6 +248,8 @@ To be released.
     different genesis block.  [[#1003], [#1004]]
  -  `Swarm<T>` became to ignore `BlockHeaderMessage` from the peers with
     different genesis block.  [[#1003], [#1004]]
+ -  The code for validating the next block had been moved to `BlockChain<T>` from
+    `BlockPolicy<T>`.  [[#1010]]
```

Comment:
```suggestion
 -  `BlockChain<T>` instead of `BlockPolicy<T>` became to validate `Block<T>`s to append
    so that even if an empty implementation of `IBlockPolicy<T>` is used `Block<T>`s are
    unable to be appended to `BlockChain<T>`.  [[#1010]]
```